frameioclient/client.py

```diff
@@ -219,6 +219,30 @@ def get_asset_children(self, asset_id, **kwargs):
     endpoint = '/assets/{}/children'.format(asset_id)
     return self._api_call('get', endpoint, kwargs)
 
+  def get_newest_assets(self, account_id, project_id, page_size):
+    """
+    Get a projects most recently added assets.
```

Comment:
`project's` <- just being pedantic here. 

This is a great add! Thanks man :)

Reply:
always mix those up :)